client/app/cloud/project/compute/loadbalancer/configure/compute-loadbalancer-configure.controller.js

```diff
@@ -0,0 +1,260 @@
+angular.module("managerApp").controller("CloudProjectComputeLoadbalancerConfigureCtrl", function ($anchorScroll, $scope, $stateParams, $q, $location, $window, $translate,  CloudProjectComputeLoadbalancerService, OvhApiIpLoadBalancing, OvhApiCloudProjectIplb, OvhApiCloudProject, ovhDocUrl, CloudMessage, IpLoadBalancerTaskService, ControllerHelper, CloudPoll, Toaster) {
```

Comment:
371 characters long line should be completly avoided

Reply:
No "use strict"? in fact by using class you would not have to include "use strict"

Reply:
ES6 please= )

Reply:
done